Library/Homebrew/formula_creator.rb

```diff
@@ -87,7 +87,11 @@ def write_formula!
 
         if @github
           @desc = @github["description"]
-          @homepage = @github["homepage"]
+          @homepage = if @github["homepage"].empty?
+            "https://github.com/#{@github["full_name"]}"
+          else
+            @github["homepage"]
+          end
```

Comment:
```suggestion
          @homepage = @github["homepage"]
          if @homepage.blank? && (github_full_name = @github["full_name"].presence)
            @homepage = "https://github.com/#{github_full_name}"
          end
```

Reply:
If `@github` is set, the `@github["full_name"]` is guaranteed to be present the same way `@github["desc"]` and other fields are there.

Reply:
Ok, if you're 100% sure.